### DISCUSSION TITLE
Copilot not suggesting unless i press buttons.

### DISCUSSION THREAD

**Discussion by Wandon12** [community/community#18314] [Copilot Conversations]
z

**Comment by dnsdudrla97** [comment]
Copilot supports custom code by entering the statement you want to think about when processing comments as follows.
<img width="413" alt="image" src="https://user-images.githubusercontent.com/17191925/173172164-c6e484c3-4b98-4245-a4f0-98323732ab8e.png">

Please check if Copilot is activated in the lower right corner. If there is a crack, it may be a state of not reinstalling and registering on github, or it may be a synchronization problem.
<img width="217" alt="image" src="https://user-images.githubusercontent.com/17191925/173172205-2a84454a-755b-4a82-8690-bde721ecec6e.png">

Please refer to the following document for more information 😄 
https://github.com/github/copilot-docs/tree/main/docs#getting-started
